lib/src/components/NxProgressBar/NxProgressBar.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { faExclamationCircle, faCheckCircle } from '@fortawesome/free-solid-svg-icons';
+import classnames from 'classnames';
+import React, { forwardRef } from 'react';
+
+import { useUniqueId } from '../../util/idUtil';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, propTypes } from './types';
+
+import './NxProgressBar.scss';
+
+const NxProgressBar = forwardRef<HTMLProgressElement, Props>(
+    function NxProgressBar(props, ref) {
+      const {
+        hasError,
+        inlineCounter = false,
```

Comment:
We generally can't use this syntax since it only considers `undefined` and not `null`.  We have to be able to handle both due to the way optional `PropTypes` work

Reply:
> We generally can't use this syntax since it only considers `undefined` and not `null`. We have to be able to handle both due to the way optional `PropTypes` work

Updated it here: 06102db95349206668c0fbd1a50442d09ea010d1